newdle/client/src/components/AvailabilityRing.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function AvailabilityRing({
+  available,
+  ifNeeded,
+  totalParticipants,
+  strokeWidth,
+  radius,
+}) {
+  const size = radius * 1.1;
+  const normalizedRadius = radius - 14;
+  const circumference = normalizedRadius * 2 * Math.PI;
+
+  return (
+    <div>
+      <svg height={size * 2} width={size * 2}>
+        <circle
+          stroke="#00ac46"
+          fill="transparent"
+          strokeWidth={strokeWidth}
+          strokeDasharray={
+            circumference * (available / totalParticipants) +
+            ' ' +
+            circumference * ((totalParticipants - available) / totalParticipants)
```

Comment:
@rpanczer i'm curious, what's the need for this calculation here? as far as I understood how this value works is that just using `circumference` here should be enough, since that way you never get more than one dash for that stroke